OneCommunityGlobal/HighestGoodNetworkApp
# keyun-fix-team-member-inactive

<img width="746" alt="Screenshot 2023-12-30 at 2 55 17 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/124097392/e4e8aed6-360c-4bda-9431-38d6766d9c64">

Video Link: https://www.loom.com/share/58f6a3eabdfe4fa384674f67f58bd297?sid=4f460680-889c-4132-872b-4e487c9ce85e 
## Related PRS (if any):
None
…

## Main changes explained:
- In allTeamsAction.js, isActive parameter was added to the addTeamMember function
- onAddUser updated to pass the isActive attribute of the user to the addTeamMember function

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner user
5. go to otherLink→ Team, and add an activate user to a team
6. verify that the user status is activate with a green circle next to the name without the page being refresh
7. Try to add an inactivate user and it will not show up in search result

## Screenshots or videos of changes:
BEFORE:
https://www.loom.com/share/58f6a3eabdfe4fa384674f67f58bd297?sid=4f460680-889c-4132-872b-4e487c9ce85e

NOW:
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/124097392/d39796b9-8b3f-45ae-9feb-18146027a20c
